pkg/node/node.go

```diff
@@ -311,6 +326,9 @@ func NewNode(
 		if apiServer != nil {
 			err = errors.Join(err, apiServer.Shutdown(ctx))
 		}
+		if err := analytics.ShutdownAnalyticsProvider(ctx); err != nil {
+			log.Trace().Err(err).Msg("failed to shutdown analytics provider")
+		}
```

Comment:
any potential race condition issues with devstack?

Reply:
addressed by moving to the server and disabling during testing.